yt_dlp/extractor/youtube.py

```diff
@@ -763,6 +793,13 @@ def _extract_video(self, renderer):
             'duration': duration,
             'view_count': view_count,
             'uploader': uploader,
+            'channel_id': channel_id,  # TODO: validate?
+            'upload_date': strftime_or_none(timestamp, '%Y%m%d'),
+            'live_status': ('is_upcoming' if scheduled_timestamp is not None else None
+                            or 'was_live' if any(i in time_text.lower() for i in ('streamed', 'premiered')) else None
+                            or 'is_live' if overlay_style is not None and overlay_style == 'LIVE' or 'live now' in badges else None),
```

Comment:
```suggestion
            'live_status': ('is_upcoming' if scheduled_timestamp is not None
                            else 'was_live' if any(i in time_text.lower() for i in ('streamed', 'premiered'))
                            else 'is_live' if overlay_style is not None and overlay_style == 'LIVE' or 'live now' in badges
                            else None),
```
or
```suggestion
            'live_status': (scheduled_timestamp is not None and 'is_upcoming'
                            or any(i in time_text.lower() for i in ('streamed', 'premiered')) and 'was_live'
                            or overlay_style is not None and overlay_style == 'LIVE' or 'live now' in badges and 'is_live'
                            or None),
```
personally, I prefer the first